src/main/java/com/byznass/tiolktrack/kernel/dao/GpsProvider.java

```diff
@@ -6,4 +6,6 @@
 public interface GpsProvider {
 
 	Gps getGpsById(String gpsId) throws NoGpsWithIdException, TiolkTrackException;
+
+	boolean exists(String gpsId) throws TiolkTrackException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.byznass.tiolktrack.kernel.TiolkTrackException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.tiolktrack.byznass.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)